front/app/containers/Admin/projects/project/analysis/TopBar/Filters/index.tsx

```diff
@@ -68,35 +72,43 @@ const Filters = ({ onClose }: FilterProps) => {
               <EmptyCustomFieldsFilter />
             </>
           )}
-          {analysis?.data.attributes.participation_method === 'ideation' && (
+          {(method?.supportsReactions ||
```

Comment:
Let's create a descriptively named const for this and put it on top? It's getting a bit hard to follow with so many conditions.